feed/Dockerfile

```diff
@@ -8,8 +8,6 @@ RUN cargo build --release
 RUN rm -rf ./src
 # Copy the project files from your machine to the container
 COPY ./ ./
-# Invalidate Cargo cache
-RUN touch -a -m ./src/main.rs
```

Comment:
Removing this line always builds dummy image

Reply:
Ok, I [found the reason](https://stackoverflow.com/questions/58473606/cache-rust-dependencies-with-docker-build#comment117943107_58474618) why I included it in the first place.
> If you're doing RUN echo "fn main() {}" > ./src/main.rs, then COPY ./src ./src, cargo may not rebuild your app. You'll need to update the last modified of the main.rs file to inform cargo to rebuild it. Before your 2nd cargo build, you need to run RUN touch -a -m ./src/main.rs to update the last modified of the file.

I'll keep this, and also think if I can switch to the stable rust and pin the base image version (currently nightly image updates frequently, which defies the caching attempts).

Reply:
You have `rust-toolchain` file that stipulates `nightly` toolchain, and the reason you do so, because it supports `#![feature(test)]`

Reply:
> You have `rust-toolchain` file that stipulates `nightly` toolchain, and the reason you do so, because it supports `#![feature(test)]`

You're right, stable rust is a no-go.